### ISSUE TITLE
Build failure version 1.9.908 on Windows XP

### ISSUE THREAD

**Issue by GoogleCodeExporter** [marutichintan/pymssql#31]
```
C:\pymssql-1.9.908>cl
Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 15.00.30729.01 for 80x86
Copyright (C) Microsoft Corporation.  All rights reserved.

usage: cl [ option... ] filename... [ /link linkoption... ]

C:\pymssql-1.9.908>cython --version
Cython version 0.13

C:\pymssql-1.9.908>python --version
Python 2.7.1

C:\pymssql-1.9.908>python setup.py build
running build
running build_ext
cythoning _mssql.pyx to _mssql.c

Error converting Pyrex file to C:
------------------------------------------------------------
...
DEF PYMSSQL_CHARSETBUFSIZE = 100
DEF MSSQLDB_MSGSIZE = 1024
DEF PYMSSQL_MSGSIZE = (MSSQLDB_MSGSIZE * 8)
DEF EXCOMM = 9

from python_version cimport PY_MAJOR_VERSION, PY_MINOR_VERSION
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:33:0: 'python_version.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
DEF PYMSSQL_CHARSETBUFSIZE = 100
DEF MSSQLDB_MSGSIZE = 1024
DEF PYMSSQL_MSGSIZE = (MSSQLDB_MSGSIZE * 8)
DEF EXCOMM = 9

from python_version cimport PY_MAJOR_VERSION, PY_MINOR_VERSION
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:33:0: 'PY_MAJOR_VERSION.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
DEF PYMSSQL_CHARSETBUFSIZE = 100
DEF MSSQLDB_MSGSIZE = 1024
DEF PYMSSQL_MSGSIZE = (MSSQLDB_MSGSIZE * 8)
DEF EXCOMM = 9

from python_version cimport PY_MAJOR_VERSION, PY_MINOR_VERSION
                           ^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:33:28: Name 'PY_MAJOR_VERSION' not declared in mod
ule 'python_version'

Error converting Pyrex file to C:
------------------------------------------------------------
...
DEF PYMSSQL_CHARSETBUFSIZE = 100
DEF MSSQLDB_MSGSIZE = 1024
DEF PYMSSQL_MSGSIZE = (MSSQLDB_MSGSIZE * 8)
DEF EXCOMM = 9

from python_version cimport PY_MAJOR_VERSION, PY_MINOR_VERSION
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:33:0: 'PY_MINOR_VERSION.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
DEF PYMSSQL_CHARSETBUFSIZE = 100
DEF MSSQLDB_MSGSIZE = 1024
DEF PYMSSQL_MSGSIZE = (MSSQLDB_MSGSIZE * 8)
DEF EXCOMM = 9

from python_version cimport PY_MAJOR_VERSION, PY_MINOR_VERSION
                                             ^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:33:46: Name 'PY_MINOR_VERSION' not declared in mod
ule 'python_version'

Error converting Pyrex file to C:
------------------------------------------------------------
...
    import uuid

import decimal
import datetime
from sqlfront cimport *
from stdio cimport fprintf, sprintf, FILE
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:41:0: 'stdio.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
    import uuid

import decimal
import datetime
from sqlfront cimport *
from stdio cimport fprintf, sprintf, FILE
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:41:0: 'fprintf.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
    import uuid

import decimal
import datetime
from sqlfront cimport *
from stdio cimport fprintf, sprintf, FILE
                  ^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:41:19: Name 'fprintf' not declared in module 'stdi
o'

Error converting Pyrex file to C:
------------------------------------------------------------
...
    import uuid

import decimal
import datetime
from sqlfront cimport *
from stdio cimport fprintf, sprintf, FILE
^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:41:0: 'sprintf.pxd' not found

Error converting Pyrex file to C:
------------------------------------------------------------
...
    import uuid

import decimal
import datetime
from sqlfront cimport *
from stdio cimport fprintf, sprintf, FILE
                           ^
------------------------------------------------------------

C:\pymssql-1.9.908\_mssql.pyx:41:28: Name 'sprintf' not declared in module 'stdi
o'

... etc, etc, the output is similar to the attachment 'output.txt' attached to 
Issue 23.
Thanks and regards,
Simon Haines.

```

Original issue reported on code.google.com by `con.amal...@gmail.com` on 6 Dec 2010 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I missed your fix for cython 0.13 (Issue 24). Please discard this bug 
report.
```

Original comment by `con.amal...@gmail.com` on 6 Dec 2010 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
Closing.
```

Original comment by `dam...@gmail.com` on 4 Jan 2011 at 7:50
- Changed state: **Invalid**
